src/Altinn.Platform/Altinn.Platform.Events/Events/Services/CosmosService.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+using Altinn.Platform.Events.Services.Interfaces;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Services
+{
+    /// <inheritdoc/>
+    [ExcludeFromCodeCoverage]
+    public class CosmosService : ICosmosService
```

Comment:
As the collection name and partition key is hard coded we should call this class something to bind it to the Events component. EventsCosmosClient or something. The alternative would be to move those hard coded values into some configuration that is injected.

Reply:
Could alternatively rename them to eventsCollectionId and eventsPartitionKey, allowing there to be added more collections into the same class in the future?

Reply:
That would lead to one service for all Cosmos interactions instead of one for each collection. That could work. Not sure about how to go about that at the moment. I think the current refactor is a step in the right direction. We can take more steps later.

Reply:
Renamed to be events specific